src/gigs-board/entity/project/kanban-view.jsx

```diff
@@ -0,0 +1,346 @@
+  edit_community_github: ({ handle, github }) =>
+    Near.call(devHubAccountId, "edit_community_github", { handle, github }) ??
+    null,
+
+  create_project: ({ author_community_handle, metadata }) =>
+    Near.call(devHubAccountId, "create_project", {
+      author_community_handle,
+      metadata,
+    }) ?? null,
+
+  update_project_metadata: ({ metadata }) =>
+    Near.call(devHubAccountId, "update_project_metadata", { metadata }) ?? null,
+
+  get_project_views_metadata: ({ project_id }) =>
+    Near.view(devHubAccountId, "get_project_views_metadata", { project_id }) ??
+    null,
+
+  create_project_view: ({ view }) =>
+    Near.call(devHubAccountId, "create_project_view", { view }) ?? null,
+
+  update_project_view: ({ view }) =>
+    Near.call(devHubAccountId, "update_project_view", { view }) ?? null,
+
+  delete_project_view: ({ id }) =>
+    Near.call(devHubAccountId, "delete_project_view", { id }) ?? null,
+
+  get_access_control_info: () =>
+    Near.view(devHubAccountId, "get_access_control_info") ?? null,
+
+  get_all_authors: () => Near.view(devHubAccountId, "get_all_authors") ?? null,
+
+  get_all_communities: () =>
+    Near.view(devHubAccountId, "get_all_communities") ?? null,
+
+  get_all_labels: () => Near.view(devHubAccountId, "get_all_labels") ?? null,
+
+  get_community: ({ handle }) =>
+    Near.view(devHubAccountId, "get_community", { handle }) ?? null,
+
+  get_post: ({ post_id }) =>
+    Near.view(devHubAccountId, "get_post", { post_id }) ?? null,
+
+  get_posts_by_author: ({ author }) =>
+    Near.view(devHubAccountId, "get_posts_by_author", { author }) ?? null,
+
+  get_posts_by_label: ({ label }) =>
+    Near.view(nearDevGovGigsContractAccountId, "get_posts_by_label", {
+      label,
+    }) ?? null,
+
+  get_root_members: () =>
+    Near.view(devHubAccountId, "get_root_members") ?? null,
+
+  useQuery: ({ name, params }) => {
+    const initialState = { data: null, error: null, isLoading: true };
+
+    const cacheState = useCache(
+      () =>
+        Near.asyncView(devHubAccountId, ["get", name].join("_"), params ?? {})
+          .then((response) => ({
+            ...initialState,
+            data: response ?? null,
+            isLoading: false,
+          }))
+          .catch((error) => ({
+            ...initialState,
+            error: props?.error ?? error,
+            isLoading: false,
+          })),
+
+      JSON.stringify({ name, params }),
+      { subscribe: true }
+    );
+
+    return cacheState === null ? initialState : cacheState;
+  },
+};
+/* END_INCLUDE: "core/adapter/dev-hub" */
+
+const configToColumns = (config) =>
+  Object.entries(config.columns).reduce((registry, [columnId, column]) => {
+    const postIds = (
+      Near.view(nearDevGovGigsContractAccountId, "get_posts_by_label", {
+        label: column.tag,
+      }) ?? []
+    ).reverse();
+
+    return {
+      ...registry,
+
+      [columnId]: {
+        ...column,
+
+        postIds:
+          tags.required.length > 0
```

Comment:
Where is the `tags` defined? I don't see it and there seems no check to format of `tags` before using it